src/Sfa.Das.Sas.Indexer.ApplicationServices/Shared/Settings/AppServiceSettings.cs

```diff
@@ -64,6 +66,13 @@ public AppServiceSettings(IProvideSettings settingsProvider)
 
         private string VstsGitBaseUrl => _settings.GetSetting("VstsGitBaseUrl");
 
+        public List<string> FrameworksExpiredRequired => GetFrameworksList(_settings.GetSetting("FrameworksExpiredRequired"));
+
+        private List<string> GetFrameworksList(string frameworkIdList)
+        {
+            return frameworkIdList.Split(',').Select(frameworkId => frameworkId.Trim()).ToList();
+        }
```

Comment:
possible null exception if there are no frameworks to add, may be change it to this?

`return string.IsNullOrEmpty(frameworkIdList) ? new List<string>() : frameworkIdList.Split(',').Select(frameworkId => frameworkId.Trim()).ToList();`